### ISSUE TITLE
Plugin found fans but says they all have lights (which they don't)

### ISSUE THREAD

**Issue by mikep8** [vivienne/homebridge-haiku-senseme#13]
Thanks for this plugin!  It was able to easily able to auto discover all of my fans but it mistakenly shows them all has having lights which they don't.  I haven't looked through the code yet but if you could point me in the right direction I'd be happy to take a look.

Thanks!

**Comment by ndpatel1280** [comment]
Hi there -- I also have this problem with my Haiku H series fan. It doesn't have a light and the log keeps throwing errors because it's expecting a light value and getting NaN instead, and it appears eventually that causes the service to crash. Anyway to disable the light?

**Comment by carsontwilson** [comment]
Is this project dead? No response on this issue for some time now...

**Comment by vivienne** [comment]
No, but I don't have access to a fan to test any of the fixes and my computer is on its last legs. I've been meaning to revisit the code but haven't had a chance yet. If anyone has updates to check in I'd be happy to merge them in though. :)

**Comment by dsully** [comment]
I'm having this issue as well. Had to downgrade to v2.1.1 in order to get the plugin to work again with fans & no lights.

**Comment by vivienne** [comment]
I think this has to do with issue #7 ... I'll have to dig back into my memory bank and take a look at what the current code is to figure out what happened. Based on the comments from the other issue the beta might have the fix, but I don't have any fans myself to test on. I don't think I had figured out a way to instantiate a fan object with or without the light service depending on the light present attribute yet.

**Comment by mikep8** [comment]
Happy to help test - none of my Haiku fans have lights but at least I can cover that test case…
